spec/lib/models/locales_updater_spec.rb

```diff
@@ -19,6 +20,7 @@
   let(:api_key) { 'test-key' }
   let(:project_id) { 'test-id' }
   let(:version) { 'test-version' }
+  let(:previous_version) { 'test-previous-version' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.